### DISCUSSION TITLE
Why does Feishin need to be connected to my router if my server is on my PC?

### DISCUSSION THREAD

**Discussion by isle9** [jeffvli/feishin#89] [General]
If I'm not connected to the internet, I can open up my Navidrome server and everything works fine, but Feishin doesn't seem to connect at all.

**Comment by jeffvli** [comment]
It's working for me from my initial tests.

I removed the default gateway from my ipv4 config so I could simulate only having access to LAN, and it still worked as expected.

**Reply by isle9** [reply]
My bad, I meant a connection to my router. My server is on my PC so I expected Feishin to still work fine.
I've changed the title.

**Reply by Zipties** [reply]
Make sure you’re using 127.0.0.1 as the IP

**Reply by isle9** [reply]
I've tried both localhost and 127.0.0.1 just to make sure, but neither work.

**Reply by Zipties** [reply]
Without more info it's hard to say then. Is it a containerized install?

**Reply by isle9** [reply]
I'm using the AppImage.
